xl_auth/grant/views.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+"""Grant views."""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from flask import Blueprint, abort, flash, redirect, render_template, url_for
+from flask_babel import lazy_gettext as _
+from flask_login import current_user, login_required
+
+from .models import Grant
+
+blueprint = Blueprint('grant', __name__, url_prefix='/grants', static_folder='../static')
+
+
+@blueprint.route('/')
+@login_required
+def home():
+    """Grant landing page."""
+    if not current_user.is_admin:
+        abort(403)
+
+    grants = Grant.query.all()
+
+    return render_template('grants/home.html', grants=grants)
+
+
+@blueprint.route('/delete/<int:id>', methods=['GET', 'DELETE'])
+@login_required
+def delete(id):
```

Comment:
<img width="334" alt="screen shot 2017-10-30 at 4 08 02 pm" src="https://user-images.githubusercontent.com/786326/32178260-980696a6-bd8c-11e7-9aaa-b6d2e646268b.png">

Same for token!